src/support/hazard.c

```diff
@@ -311,15 +311,9 @@ __wt_hazard_check(WT_SESSION_IMPL *session, WT_REF *ref, WT_SESSION_IMPL **sessi
      */
     __wt_session_gen_enter(session, WT_GEN_HAZARD);
 
-    /*
-     * No lock is required because the session array is fixed size, but it may contain inactive
-     * entries. We must review any active session that might contain a hazard pointer, so insert a
-     * read barrier after reading the active session count. That way, no matter what sessions come
-     * or go, we'll check the slots for all of the sessions that could have been active when we
-     * started our check.
-     */
-    WT_ORDERED_READ(session_cnt, conn->session_cnt);
-    for (s = conn->sessions, i = max = walk_cnt = 0; i < session_cnt; ++s, ++i) {
+    max = walk_cnt = 0;
```

Comment:
Should initialize these at the start of the function.

Reply:
Done in [f99dc00](https://github.com/wiredtiger/wiredtiger/pull/9295/commits/f99dc009fef7049dcc3c7971da1a43bf451be11a)